test/engine.test.ts

```diff
@@ -388,4 +485,46 @@ describe('Engine test', () => {
       expect(result).toEqual(false);
     });
   });
+
+  describe('Vector condition v2', () => {
```

Comment:
What is v2?

Reply:
A version bump on the audience definition -- I'll update the title to make it clearer